ProcessMaker/Traits/ExtendedPMQL.php

```diff
@@ -135,20 +135,32 @@ private function parseValue($expression)
         }
 
         // Check to see if the value is parsable as a date
-        if ($value instanceof IntervalExpression || (is_string($value) && strlen($value) > 1)) {
-            try {
-                $parsed = Carbon::parse($value, auth()->user()->timezone);
-                if ($parsed->isMidnight()) {
-                    $value = $parsed->toDateString();
-                } else {
-                    $parsed->setTimezone(config('app.timezone'));
-                    $value = $parsed->toDateTimeString();
+        if ((is_string($value) && strlen($value) > 1)) {
+            switch ($value) {
+                case $value instanceof IntervalExpression: 
+                    $value = $this->parseDate($value);
+                    break;
+                default: 
+                    // Check to see if the value is a date/datetime formatted if not return original value
+                    $isDateFormatted = Carbon::hasFormatWithModifiers($value, 'Y#m#d');
+                    $isDateTimeFormatted = Carbon::hasFormatWithModifiers($value, 'Y#m#d h:i:s');
```

Comment:
@sanjacornelius I think we should use `H` here instead of `h` since we're using a 24 hour clock in other places and the format doesn't specify AM or PM

Reply:
@nolanpro  Good catch! Code has been updated thank you!